config/configStruct/base.go

```diff
@@ -3,11 +3,11 @@ package configStruct
 import "fmt"
 
 type Base struct {
-	Name     string `env:"SERVER_NAME" envDefault:"unknown"`
-	Address  string `env:"SERVER_ADDRESS" envDefault:"localhost"`
-	Port     int    `env:"SERVER_PORT" envDefault:"8080"`
-	NameCode int32  `env:"NAME_CODE" envDefault:"0"`
-	NodeCode int32  `env:"NODE_CODE" envDefault:"0"`
+	Name     string `mapstructure:"Name" default:"unknown" `
```

Comment:
If we delete tags which named `env`, we will can't load it from environment variables

Reply:
I use viper to get those environment variables so 'env' tag is useless here